virt/managing_vms/virt-managing-vms-openshift-pipelines.adoc

```diff
@@ -43,6 +44,27 @@ include::modules/virt-running-ssp-pipeline-web.adoc[leveloffset=+2]
 
 include::modules/virt-running-ssp-pipeline-cli.adoc[leveloffset=+2]
 
+[id="virt-deprecated-tasks_{context}"]
+== Removing deprecated or unused resources
+
+* You can clean up Tekton-Tasks-Operator resources associated with the operator.
+
+.Procedure
+
+. To remove any remaining Tekton-related resources from the cluster, run the following command:
+
```

Comment:
Add a plus sign here so that the code block appears indented correctly under Step 1.